### PR TITLE
feat(workflow): Clickable "Select a Project"

### PR DESCRIPTION
**Pull request by MaxBittker**
make entire select a project button clickable when it displays this default message (not just the caret)
screenshot 2018-01-11 13 42 56

### CODE REVIEW

**Comment by MaxBittker** [comment]
if this isn't something we want because of inconsistency, i'm ok with that

File: src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx [hunk 1]
```diff
@@ -330,9 +330,13 @@ const ProjectSelector = createReactClass({
           <Link to={`/${org.slug}/`} className="home-crumb">
             <span className="icon-home" />
           </Link>
-          {this.state.activeProject
-            ? this.getLinkNode(this.state.activeTeam, this.state.activeProject)
-            : t('Select a project')}
+          {this.state.activeProject ? (
+            this.getLinkNode(this.state.activeTeam, this.state.activeProject)
+          ) : (
+            <span onClick={() => (this.state.isOpen ? this.onClose() : this.onOpen())}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx]: We should use an unstyled Button (does that exist?) - or at least make sure it has `role="button"` and `cursor: pointer`

**Review by billyvg — APPROVED** [review]
I'm neutral on this - on one hand not having an action associated with "Select a Project" can be frustrating, on the other, it's inconsistent with the behavior when there is a project selected. I think I lean towards having it open, but up to you

**Review by ckj — APPROVED** [review]
> if this isn't something we want because of inconsistency, i'm ok with that

@MaxBittker I think this is one of the few places it's better to be "inconsistent." I'm 👍 on merging.

File: src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx (hunk 1 above)
Comment (MaxBittker) [reply src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx]: good catch, will do

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Pull request merged**